Public/Src/Engine/Dll/Engine.cs

```diff
@@ -1113,6 +1113,9 @@ public static void PopulateLoggingAndLayoutConfiguration(CommandLineConfiguratio
                 mutableConfig.Schedule.EarlyWorkerRelease = false;
             }
 
+            PathTranslator.CreateIfEnabled(mutableConfig.Logging.SubstSource, mutableConfig.Logging.SubstTarget, pathTable, out var translator);
+            mutableConfig.Sandbox.GlobalUnsafeUntrackedScopes = mutableConfig.Sandbox.GlobalUnsafeUntrackedScopes.Select(path => AbsolutePath.Create(pathTable, translator.Translate(path.ToString(pathTable)))).ToList();

```

Comment:
We don't want to perform translation here. The paths given in the `GlobalUnsafeUntrackedScopes` should contain those specified in the command line argument, and unmodified.

The translated version should be given at the sandboxed process layer. At that layer, we have a directory translator that is embedded in the file access manifest. You can use that translator to prepare the paths.